coalib/parsing/DefaultArgParser.py

```diff
@@ -81,6 +81,18 @@ def default_arg_parser(formatter_class=None):
                             help='Files that will be analyzed will be '
                                  'restricted to those in the globs listed '
                                  'in this argument as well the files setting')
+    arg_parser.add_argument('-C',
+                            '--caching',
+                            nargs='?',
+                            const=True,
+                            metavar='BOOL',
+                            help='Run coala only on files that have changed '
+                                 'since the last time coala was run')
```

Comment:
can we mention "Caching is experimental and will be enabled by default by the next release. This option will be removed."?